From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

Commit:
update sovereign clouds breadcrumbs

v-prhal committed:
Merge pull request #1 from MicrosoftDocs/master

Sync with master

v-prhal committed:
1074237 - "Capacity limits" removed from the Language list.

ktoliver committed:
edit pass

ktoliver committed:
edits

Commit:
update breadcrumb names

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

Commit:
Updating volume driver repo

Commit:
initial changes

kevinlam1 committed:
Update logic-apps-pricing.md

Updating to include new billing model for connectors and the basic integration account and removing App Service references.

kevinlam1 committed:
Update logic-apps-pricing.md

Fixed typos

kevinlam1 committed:
Update logic-apps-pricing.md

Addressed acrolinx feedback

Commit:
draft sections with code

Commit:
change utterances column to request

Commit:
ui image

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

curtand committed:
rewrite classic portal content

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

ktoliver committed:
more edits

ktoliver committed:
more edits

Commit:
Updating typos and adding ipv6 requirement for mac

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Updating typos and adding ipv6 requirement for mac

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Revert "Updating typos and adding ipv6 requirement for mac"

This reverts commit c406617eab9b2c707b6a882e9140bcc327024ddd.

Commit:
Reverting docker for mac changes

Commit:
Merge branch t push origin
'master' of https://github.com/Microsoft/azure-docs-pr

tamram committed:
changing headings per feedback

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
Updating volume driver info

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
SaaS Tutorial - Clear Review

tamram committed:
added link to samples page

Commit:
code changes

Commit:
output and next steps

Commit:
Toc title

Commit:
links

Commit:
use direct link to sample code

Commit:
fix typos

Commit:
headings and next step

ktoliver committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-sync-files-troubleshoot

ktoliver committed:
fix Windows Server instances

ktoliver committed:
fix link

Commit:
remove comments

Commit:
fix title

Commit:
Added not about public IPs.

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Added public IP note.

chetansriv committed:
SaaS Tutorial : Apex Portal

Commit:
fix incorrect links

rwike77 committed:
removing article

rwike77 committed:
created redirect

rwike77 committed:
updated inbound links

chetansriv committed:
Apex Portal update

Commit:
Fix typos

spelluru committed:
fix description

Commit:
Fix image

Commit:
fix broken syntax highlight

KumudD committed:
added link to Azure DNS doc

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into DNSUpdate1110

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into replica1108

ShawnJackson committed:
Update storage-sync-files-deployment-guide.md

ShawnJackson committed:
Update storage-sync-files-troubleshoot.md

rnagpal committed:
Release Notes for .NET SDK 1.19.0

@mimig1 @kirankumarkolli : Please review. I'll add the sign off tag.

rnagpal committed:
Release Notes for .NET Core SDK 1.7.0

@mimig1 @kirankumarkolli : Please review and I'll add sign off tag

motanv committed:
RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter information and add release note

rwike77 committed:
removing duplicates

PRMerger10 committed:
Merge pull request #27503 from KumudD/DNSUpdate1110

Added Azure DNS doc link

v-thepet committed:
fix todays broken links

Lisaco88 committed:
Merge pull request #27332 from mani-ramaswamy/master

Fixing typos for networking mode and adding ipv6 requirement

PRMerger13 committed:
Merge pull request #27506 from rnagpal/patch-19

Release Notes for .NET Core SDK 1.7.0

PRMerger10 committed:
Merge pull request #27505 from rnagpal/patch-18

Release Notes for .NET SDK 1.19.0

v-thepet committed:
try to do it without involving art

rjagiewich committed:
Merge pull request #27393 from chetansriv/clearreview

Clearreview

ShawnJackson committed:
Update storage-sync-files-troubleshoot.md

rjagiewich committed:
Merge pull request #27468 from chetansriv/apexportal

Apexportal

tfitzmac committed:
deleted naming convention

PRMerger19 committed:
Merge pull request #27510 from motanv/chaos_article_update_filter

RDTask 10625124 : Update Chaos ACOM article with Chaos Target Filter …

ShawnJackson committed:
Merge pull request #26673 from ktoliver/storage-sync-files-troubleshoot

edit pass: TFS 72116 (4 files)

Lisaco88 committed:
Merge pull request #27489 from rwike77/delete

Delete article, create redirect, update inbound links

Commit:
Merge pull request #27513 from v-thepet/newfix

production bug: fix today's broken links

Lisaco88 committed:
Merge pull request #27511 from rwike77/tocupdate

removing duplicates

tamram committed:
fixing link

Commit:
Merge pull request #27492 from spelluru/fixpartition1110

fix description

Commit:
devices02

tamram committed:
Update storage-quickstart-blobs-java.md

Commit:
Merge pull request #27547 from MarkusVi/devices02

devices02

PRMerger12 committed:
Merge pull request #27351 from tamram/tamram-1109

changing headings per feedback

craigshoemaker committed:
fix typo

craigshoemaker committed:
reassign node.js articles from tarcher to cshoe

PRMerger15 committed:
Merge pull request #27554 from craigshoemaker/crs-devquickstarts-webapi-nodejs-fix-typo

fix typo

kevinlam1 committed:
Update logic-apps-pricing.md

Updated from review feedback.

Commit:
Merge pull request #27549 from MicrosoftDocs/tamram-patch-6

Update storage-quickstart-blobs-java.md

seanmck committed:
Updating docs to point to eastus rather than westus2 due to temporary capacity issues

tamram committed:
fixing indent

Commit:
Merge pull request #27557 from tamram/tamram-1111

fixing indent

Commit:
catechref02

PRMerger13 committed:
Merge pull request #27560 from MarkusVi/catechref02

catechref02

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into replica1108

MikeDodaro committed:
Fix typos in quickstart headings: Image -> Web search

rnagpal committed:
Update Release Notes for 1.19.0

rnagpal committed:
Update Release Notes for .NET Core SDK 1.7.0

@mimig1 : I'll sign off after you scrub this.

PRMerger16 committed:
Merge pull request #27555 from craigshoemaker/crs-reassign-node-articles

reassign node.js articles from tarcher to cshoe

PRMerger10 committed:
Merge pull request #27568 from rnagpal/patch-21

Update Release Notes for .NET Core SDK 1.7.0

rnagpal committed:
Update documentdb-sdk-dotnet.md

PRMerger19 committed:
Merge pull request #27567 from rnagpal/patch-20

Update Release Notes for 1.19.0

gsacavdm committed:
Update Company Branding instructions to new portal

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into replica1108

curtand committed:
Pointed to current content from vnet and vm teams

GitHubber17 committed:
Merge pull request #27518 from tfitzmac/1110namepattern

deleted naming convention

curtand committed:
Acrolinx

curtand committed:
SEO

MGoedtel committed:
New article for collecting auzre PaaS

curtand committed:
final updating to Azure portal

MGoedtel committed:
updated TOC

MGoedtel committed:
updated intro paragraph

MGoedtel committed:
updated image to obscure IDs

GitHubber17 committed:
Merge pull request #27152 from kevinlam1/patch-14

Update logic-apps-pricing.md

curtand committed:
final updating to Azure portal

GitHubber17 committed:
Merge pull request #27428 from tamram/tamram-1110

added link to samples page

curtand committed:
link fix

spelluru committed:
wrong text in Sink

MGoedtel committed:
improved details in config steps

tfosmark committed:
Merge pull request #27576 from curtand/replica1108

Pointing classic content to current VM and vnet procedures.

PRMerger19 committed:
Merge pull request #27556 from seanmck/aks-regionredirect

Updating docs to point to eastus rather than westus2 due to temporary…

Commit:
Merge pull request #27571 from gsacavdm/patch-2

Update Company Branding instructions to new portal

Commit:
Update iot-hub-arduino-iot-devkit-az3166-get-started.md

linda33wj committed:
Update perf default behavior

PRMerger20 committed:
Merge pull request #27603 from shizn/master

add simulator link

Commit:
Merge pull request #27605 from linda33wj/master

Update perf default behavior

nsoneji committed:
Added vCenter 6.5, Windows 2016 support. Corrected few support matrix.

dominicbetts committed:
Address Livefyre comments

dominicbetts committed:
Fix typo

PRMerger16 committed:
Merge pull request #27607 from nsoneji/master

 Added vCenter 6.5, Windows 2016 support. Corrected few support matrix.

PRMerger15 committed:
Merge pull request #27608 from dominicbetts/suite-livefyre

Address Livefyre comments

PRMerger19 committed:
Merge pull request #27609 from dominicbetts/suite-livefyre

Fix typo

Commit:
Update iot-hub-python-getstarted.md

PRMerger16 committed:
Merge pull request #27611 from shizn/master

Add UA in python sim device tutoral

Commit:
taking ownership in preparation to remove per engineering request

Commit:
Merge pull request #27465 from jimdial/patch-31

Added not about public IPs.

Commit:
Merge pull request #27467 from jimdial/patch-32

Added public IP note.

PRMerger20 committed:
Merge pull request #27612 from Blackmist/fresh5

taking ownership in preparation to remove per engineering request

Commit:
Merge pull request #27580 from spelluru/adftablefix1112

wrong text in Sink

spelluru committed:
fixed date -> scheduledTime

Commit:
freshness

spelluru committed:
fixed issues

Commit:
Merge pull request #27618 from Blackmist/fresh5

freshness

Commit:
Merge pull request #27617 from spelluru/partitionfix1113

fixed date -> scheduledTime

dominicbetts committed:
Update link to most recent version of standard

ktoliver committed:
Merge pull request #26807 from v-savila/1102387

Add breadcrumb for sovereign clouds

MGoedtel committed:
updated article per PR reviewer

PRMerger15 committed:
Merge pull request #27566 from MikeDodaro/correct-headings-quickstarts

Fix typos in quickstart headings: Image search -> Web search

Lisaco88 committed:
Merge pull request #27577 from MGoedtel/NewArticle1112

New article for collecting auzre PaaS

ktoliver committed:
Merge pull request #26543 from v-prhal/master

 1074237 - "Capacity limits" removed from the Language list.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger14 committed:
Merge pull request #27622 from dominicbetts/amqp-cbs-link

Update link to most recent version of standard

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

Lisaco88 committed:
Merge pull request #27095 from DeniseMak/11-6-tutorial

tutorial revisions - WIP

v-alje committed:
Merge pull request #27624 from MicrosoftDocs/FromPublicRepo

From public repo
